File: lab 13-14/model/bike.py
from equipment import Equipment

class Bike(Equipment):

    def __init__(self, produce_name, weight, producer, gear_numbers, number_of_brake, type):
        super(Bike, self).__init__(produce_name, weight,type)
        self.producer = producer
        self.gear_numbers = gear_numbers
        self.number_of_brake = number_of_brake

    def __str__(self):
        return super(Bike, self).__str__() + ', producer = ' + str(self.producer) +\
               ', gear_numbers = ' + str(self.number_of_brake)

File: Lab 5/work/src/main/java/ua/lviv/iot/Bike.java
package ua.lviv.iot;

public class Bike extends Equipment {

    private int gearsNumber;
    private int numberOfBrake;

    public Bike(String ProduceName, double weight, SportType sportType, int gearsNumber, int numberOfBrake) {
        super(ProduceName, weight, sportType);
        this.gearsNumber = gearsNumber;
        this.numberOfBrake = numberOfBrake;
    }

    public int getGearsNumber() {
        return gearsNumber;
    }

    public void setGearsNumber(int gearsNumber) {
        this.gearsNumber = gearsNumber;
    }

    public int getNumberOfBrake() {
        return numberOfBrake;
    }

    public void setNumberOfBrake(int numberOfBrake) {
        this.numberOfBrake = numberOfBrake;
    }

    @Override
    public String toString() {
        return "Bike{" +
                "gearsNumber = " + gearsNumber +
                ", numberOfBrake = " + numberOfBrake + " " +
                super.toString() +
                "}\n";
    }
}